TCAT/Controllers/RouteDetailViewController.swift

```diff
@@ -479,7 +488,7 @@ class RouteDetailViewController: UIViewController, GMSMapViewDelegate, CLLocatio
         // Place and format bottom summary label
         let summaryBottomLabel = UILabel()
         if let totalTime = Time.dateComponents(from: route.departureTime, to: route.arrivalTime).minute {
-            summaryBottomLabel.text = "Trip Duration: \(abs(totalTime)) minutes"
+            summaryBottomLabel.text = "Trip Duration: \(abs(totalTime)) minute\(totalTime == 1 ? "" : "s")"
```

Comment:
thanks for determining between 1 minute and 2 minutes. I dislike when apps say 1 minutes

Reply:
💙